src/zino/statemodels.py

```diff
@@ -75,11 +109,8 @@ class DeviceState(BaseModel):
     ports: Dict[int, Port] = {}
     alarms: Optional[Dict[AlarmType, int]] = None
     boot_time: Optional[datetime.datetime] = None
-    bgp_style: Optional[BgpStyle] = None
-    bgp_peers: Optional[List[IPAddress]] = []
-    bgp_peer_up_times: Optional[Dict[IPAddress, int]] = dict()
-    bgp_peer_admin_status: Optional[Dict[IPAddress, BgpAdminStatus]] = dict()
-    bgp_peer_oper_states: Optional[Dict[IPAddress, BgpOperState]] = dict()
+    bgp_style: Optional[BGPStyle] = None
+    bgp_peers: Optional[dict[IPAddress, BGPPeerSession]] = dict()
```

Comment:
"Optional" means that `bgp_peers` may be `None`, but in the same line you set a default of "dict()" which means it doesn't start out as None. Does a value "None" make sense, considering that `bool(dict()) is bool(None)`, that is: both falsey?